sklearn/manifold/_barnes_hut_tsne.pyx

```diff
@@ -0,0 +1,626 @@
+# cython: boundscheck=False
```

Comment:
I have never seen that before. Does it do the same as `@cython.boundscheck(False)` for each method?